docs/docs/faq/question_07.md

```diff
@@ -0,0 +1,24 @@
+
+# Can I use a custom prior with sbi?
+
+Yes, if you pass a class that mimics the behaviour of a torch distribution, then sbi will wrap it as a torch distribution and can use it from there.
+
+Essentially, the class needs two methods:
+- `.sample(sample_shape)`, where sample_shape is a shape tuple, e.g., `(m,n)`/`(n,)`, and returns m batches of n samples, e.g., of shape `(m, n, 2)`/`(n, 2)` for a two dimenional prior.
```

Comment:
Let's avoid the case of 2D samples here. It's not an essential feature. Let's just use the `(n,)` case